# python3 is used for running tests, not for building or running.
#
# No generalized MacPorts mechanism exists as of 29-JAN-2025 for requiring a
# minimum Python version, so we just disable the cmake configuration
# requirement.
#
# See <https://trac.macports.org/ticket/71968#comment:5>

--- CMakeLists.txt.orig	2025-01-29 15:08:44
+++ CMakeLists.txt	2025-01-29 15:11:06
@@ -668,7 +668,7 @@
   find_package(Threads REQUIRED)
 
   # Python 3.5 is the version shipped in Ubuntu Xenial
-  find_package(Python3 3.5 REQUIRED COMPONENTS Interpreter)
+#  find_package(Python3 3.5 REQUIRED COMPONENTS Interpreter)
 
   if (NOT USE_SYSTEM_GTEST)
     if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/third_party/gtest/googletest)
